go-controller/pkg/clustermanager/clustermanager.go

```diff
@@ -254,6 +277,28 @@ func (cm *ClusterManager) StartClusterManager() error {
 		}
 	}
 
+	klog.Infof("%v", config.OVNKubernetesFeature.EnableEgressIP)
+	if config.OVNKubernetesFeature.EnableEgressIP {
```

Comment:
Does this need to be moved to `Run` function actually? after `WatchNodes()` ? that looks like the spot to do it?

Reply:
also update `hasResourceAnUpdateFunc`

Reply:
move metrics as well!